data_loaders/README.md

```diff
@@ -0,0 +1,52 @@
+# Data Loaders
+
+## 1. Corona Data Scraper
+
+
+To quote the [Corona Data Scraper](https://coronadatascraper.com) web page,
+
+> Corona Data Scraper pulls COVID-19 Coronavirus case data from verified sources, finds the corresponding GeoJSON features, and adds population data.
+
+We transform this data for use in the Covasim parameter format. It is stored
+in CSV-format. 
+
+### Updating
+
+To update the  Corona Data Scraper data,
+
+```bash
+python data_loaders/load_corona_data_scraper_data.py 
```

Comment:
I was considering what a unified single file that could look like here: https://github.com/InstituteforDiseaseModeling/covasim/blob/e1707f931011ca3c6bf72598f3c371ec53036259/covasim/datasets/update_sources.py